runtime/gc_vlhgc/ContinuationObjectBufferVLHGC.cpp

```diff
@@ -129,9 +130,11 @@ MM_ContinuationObjectBufferVLHGC::iterateAllContinuationObjects(MM_EnvironmentBa
 					omrobjectptr_t object = region->getContinuationObjectList()->getHeadOfList();
 					while (NULL != object) {
 						Assert_MM_true(region->isAddressInRegion(object));
+						envVLHGC->_continuationStats._continuationAlive += 1;
 						omrobjectptr_t next = extensions->accessBarrier->getContinuationLink(object);
 						J9VMContinuation *continuation = J9VMJDKINTERNALVMCONTINUATION_VMREF((J9VMThread *)env->getLanguageVMThread(), object);
 						if (NULL != continuation) {
+							envVLHGC->_continuationStats._continuationStarted += 1;
```

Comment:
This is somewhat reasonable, if we assume we want to trace only active ones (started, but not finished) and since VMRef is set to null soon after the continuation is finished.
Alternatively we could look at started/finished flags in the java object (in GC state field), what would probably be formally more precise, but marginally different.
@babsingh and @fengxue-IS , any preference on how we determine started (and if we want to call them started vs active vs anything else)?

Reply:
Since the purpose of this is to monitor continuation usage in the VM, I think its fine to track what VM/GC actually cares rather than what the java field is set to. This should not have any impact on the data in reality.

Reply:
I was referring to our/VM hidden field 'state' (not Class lib fields), which to me is the primary info. If for some reason a continuation finished, but freeing the native continuation struct was delayed or intentionally deferred (for example, for caching purposes), we would incorrectly concluded that the continuation is still active. 

I'd actually prefer to base it on the state (rather than continuation VMREF). We already have an API VM_ContinuationHelpers::isActive() that relies on that state field, and we use it in GC when deciding if we need to scan  the stack or not. 

I'd argue that even the existing code (for the purpose of JIT code cache reclamation) should have relied on the state (then, we could assert that continuation VMREF is non null).

In practice, the difference is probably negligibe, though. We do seem to free the native struct and reset continuation VMREF soon after changing the state field. 

Still remains the dilemma if we should call it active or started. Again, I prefer to call it 'active', but I'll let it be 'started' based on this comment: https://github.com/eclipse/omr/pull/7386#discussion_r1646558951

Reply:
I don't have anything against using the `VM_ContinuationHelpers::isActive()` API, there doesn't seem to be any difference in terms of counting result (vm access is retained between updating state and setting the VMREF)